mne/viz/circle.py

```diff
@@ -424,3 +425,77 @@ def plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
 
     plt_show(show)
     return fig, axes
+
+
+def plot_channel_labels_circle(labels, colors=None, picks=None, **kwargs):
+    """Plot labels for each channel in a circle plot.
```

Comment:
Can you say here what is the rationale for this function. I would suggest to say it makes sense for sEEG channels a not any type of channels.

Reply:
I think you could use it for any channel type provided you have some labels/classes associated with it but I agree there are not a lot of other use cases.

Reply:
Well the pb I see is that field propagation is not isotropic  ie distance to source is not monotic with field strength. I fear such a function without a warning can enourage the type of false conclusions you can see in certain EEG papers.